File: ui/src/main/java/com/midtrans/sdk/corekit/models/BillingAddress.java
package com.midtrans.sdk.corekit.models;

public class BillingAddress extends com.midtrans.sdk.uikit.api.model.Address {
    public BillingAddress(String firstName, String lastName, String address, String city, String postalCode, String phone, String countryCode) {
        super(firstName, lastName, address, city, postalCode, phone, countryCode);
    }
}

File: corekit/src/main/kotlin/com/midtrans/sdk/corekit/api/model/BillingAddress.kt
package com.midtrans.sdk.corekit.api.model

data class BillingAddress(
    val firstName: String? = null,
    val lastName: String? = null,
    val address: String? = null,
    val city: String? = null,
    val postalCode: String? = null,
    val phone: String? = null,
    val countryCode: String? = null
)
